### ISSUE TITLE
de-compiling vpn support

### ISSUE THREAD

**Issue by gothmog123** [andrew-bibb/cmst#145]
Hi keep up the great work - would it be possible to add a switch disabling vpn support at compile time? I feel like openvpn works best in its command line client than anywhere else. It's experimental even in NM.

**Comment by andrew-bibb** [comment]
There is a new commandline option to diasable VPN support.  It was put in about two and a half weeks ago so it only exists in the current code base, not yet in an official release.  Command line options are -n (--disable-vpn).  When started with these options all VPN controls in CMST will be disabled and the program will bypass connecting to the connman-vpn daemon. 

Doing it at build time currently would not be completely possible. The program is not setup to be modular like that.  There are parts that I could isolate out, but not all of it (mainly the UI page).  The question however does interest me so it may be something I look at doing when I pick up major programming again in the Fall (we have a very short summer in these parts and I try not to spend it sitting at a computer). 

The Connman VPN stuff supports more than just OpenVPN, although Connman does list all of the VPN support as experimental.  After the issues with the counters (also labeled experimental) I had decided I was not going to support any more Connman items marked that way.  I changed my mind on this one feature mainly mainly because it was asked for. 

Let me know if the commandline switch will be sufficient for your needs.  If not I'll see what I can do about rearranging the code to implement it.  Likely won't be until later in the year however.
